txpipe/utils/patches.py

```diff
@@ -241,23 +241,24 @@ def run(cls, cat, chunk_rows, comm=None):
                 print(
                     f"Catalog {cat.file_name} does not have a patch directory set, so not making patches."
                 )
-            return
+            return 0
```

Comment:
It should probably also be mentioned in the documentation for this class that it now returns a scalar value!